src/tables/math.rs

```diff
@@ -0,0 +1,1062 @@
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Standard shift up applied to the top element of a stack in display style.
+    #[inline]
+    pub fn stack_top_display_style_shift_up(&self) -> MathValueRecord {
+        self.data.get(STACK_TOP_DISPLAY_STYLE_SHIFT_UP_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Standard shift down applied to the bottom element of a stack. Positive for moving in the
+    /// downward direction.
+    #[inline]
+    pub fn stack_bottom_shift_down(&self) -> MathValueRecord {
+        self.data.get(STACK_BOTTOM_SHIFT_DOWN_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Standard shift down applied to the bottom element of a stack in display style. Positive
+    /// for moving in the downward direction.
+    #[inline]
+    pub fn stack_bottom_display_style_shift_down(&self) -> MathValueRecord {
+        self.data.get(STACK_BOTTOM_DISPLAY_STYLE_SHIFT_DOWN_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Minimum gap between (ink) bottom of the top element of a stack, and the (ink) top of the
+    /// bottom element. Suggested: 3 × default rule thickness.
+    #[inline]
+    pub fn stack_gap_min(&self) -> MathValueRecord {
+        self.data.get(STACK_GAP_MIN_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Minimum gap between (ink) bottom of the top element of a stack, and the (ink) top of the
+    /// bottom element in display style. Suggested: 7 × default rule thickness.
+    #[inline]
+    pub fn stack_display_style_gap_min(&self) -> MathValueRecord {
+        self.data.get(STACK_DISPLAY_STYLE_GAP_MIN_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Standard shift up applied to the top element of the stretch stack.
+    #[inline]
+    pub fn stretch_stack_top_shift_up(&self) -> MathValueRecord {
+        self.data.get(STRETCH_STACK_TOP_SHIFT_UP_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Standard shift down applied to the bottom element of the stretch stack. Positive for
+    /// moving in the downward direction.
+    #[inline]
+    pub fn stretch_stack_bottom_shift_down(&self) -> MathValueRecord {
+        self.data.get(STRETCH_STACK_BOTTOM_SHIFT_DOWN_OFFSET..)
+            .and_then(|data| MathValueRecord::parse(data, self.data))
+            .unwrap_or_default()
+    }
+
+    /// Minimum gap between the ink of the stretched element, and the (ink) bottom of the element
+    /// above. Suggested: same value as upperLimitGapMin.
```

Comment:
I would suggest using method reference instead of `upperLimitGapMin`, i.e. `[upper_limit_gap_min]` (or maybe `MathConstants::upper_limit_gap_min` to satisfy auto-linker, you can check it using `cargo doc` locally).